system_metrics_collector/test/system_metrics_collector/test_periodic_measurement_node.cpp

```diff
@@ -61,6 +61,8 @@ class TestPeriodicMeasurementNode : public ::system_metrics_collector::PeriodicM
     return static_cast<double>(sum.load());
   }
 
+  void publishStatisticMessage() {}
```

Comment:
```suggestion
  void publishStatisticMessage() override {}
```

(not sure)

Reply:
This is just there for the build to pass. It serves no functional purpose in this unit test.

I put the `override` keyword there anyway.